packages/neutrino-preset-web/index.js

```diff
@@ -67,15 +67,22 @@ module.exports = (neutrino) => {
   neutrino.use(imageLoader);
   neutrino.use(htmlTemplate, neutrino.options.html);
   neutrino.use(namedModules);
+
+  neutrino.config.entry('index').prepend(require.resolve('nodent-runtime'));
```

Comment:
We are already creating the `index` entry in this file on line 101, so you can change it to:

```js
.entry('index')
  .add(require.resolve('nodent-runtime'))
  .add(neutrino.options.entry)
```